### PR TITLE
Merge v0.24.1

### PR DESCRIPTION
**Pull request by schancel**

### CODE REVIEW

File: contrib/aur/bitcoin-abc-qt/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Maintainer: Josh Ellithorpe <quest@mac.com>
```
Comment (schancel) [inline contrib/aur/bitcoin-abc-qt/PKGBUILD:1]: Delete

File: src/test/cuckoocache_tests.cpp [hunk 2]
```diff
@@ -7,6 +7,8 @@
 #include <random.h>
 #include <script/sigcache.h>
 
+#include <deque>
```
Comment (schancel) [inline src/test/cuckoocache_tests.cpp:10]: ???

File: src/wallet/test/wallet_tests.cpp [hunk 3]
```diff
@@ -866,4 +866,42 @@ BOOST_FIXTURE_TEST_CASE(CreateWalletFromFile, TestChain100Setup) {
...
+    CKey key;
+    key.MakeNewKey(true);
+    AddKey(*wallet, key);
+
+    std::string error;
+    m_coinbase_txns.push_back(
+        CreateAndProcessBlock({},
+                              GetScriptForRawPubKey(coinbaseKey.GetPubKey()))
+            .vtx[0]);
+    auto block_tx = TestSimpleSpend(*m_coinbase_txns[0], 0, coinbaseKey,
```
Comment (schancel) [inline src/wallet/test/wallet_tests.cpp:881]: Needs spend index updated.

File: src/qt/forms/overviewpage.ui [hunk 4]
```diff
@@ -282,7 +287,7 @@
                <string>Your current total balance</string>
               </property>
               <property name="text">
-               <string notr="true">0.000 000 00 BCH</string>
+               <string notr="true">21 000 000 000 000.00 XPI</string>
```
Comment (EyeOfPython) [inline src/qt/forms/overviewpage.ui:290]: I assume this should be `2 100 000 000.000 000 XPI`?

File: src/test/intmath_tests.cpp [hunk 5]
```diff
@@ -1,61 +1,146 @@
...
+#include <test/lcg.h>
 #include <test/util/setup_common.h>
 
+#include <script/intmath.h>
+
+#include <boost/multiprecision/cpp_int.hpp>
+
 BOOST_FIXTURE_TEST_SUITE(intmath_tests, BasicTestingSetup)
 
-BOOST_AUTO_TEST_CASE(log2fixedpoint_tests) {
```
Comment (EyeOfPython) [inline src/test/intmath_tests.cpp:10]: Where did this go? I can't find it. Best is probably to merge both files

**Review by EyeOfPython — CHANGES_REQUESTED** [review]

File: src/test/intmath_tests.cpp (hunk 5 above)
Comment (schancel) [reply src/test/intmath_tests.cpp:10]: Yeah, oops.

File: src/qt/forms/sendcoinsdialog.ui [hunk 6]
```diff
@@ -1137,7 +1137,7 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
-          <string notr="true">123.456 BCH</string>
+          <string notr="true">123.456 XEC</string>
```
[the remark this answers is not in the document]
Comment (schancel) [reply src/qt/forms/sendcoinsdialog.ui:1140]: Wasn't even paying attention. Figured they were done with the rebrand on this one.

File: test/functional/rpc_createmultisig.py [hunk 7]
```diff
@@ -3,6 +3,11 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test multisig RPCs"""
+import binascii
+import decimal
+import itertools
+import json
+import os
 
 import binascii
```
Comment (EyeOfPython) [inline test/functional/rpc_createmultisig.py:12]: Double imports?

Comment (EyeOfPython) [reply test/functional/rpc_createmultisig.py:12]: Something didn't quite work here

File: test/functional/test_runner.py [hunk 8]
```diff
@@ -354,13 +354,15 @@ def run_tests(test_list, build_dir, tests_dir, junitoutput, tmpdir, num_jobs, te
               enable_coverage=False, args=None, combined_logs_len=0, build_timings=None, failfast=False):
     args = args or []
 
-    # Warn if lotusd is already running (unix only)
+    # Warn if bitcoind is already running
```
Comment (EyeOfPython) [inline test/functional/test_runner.py:357]: keep lotusd

**Review by EyeOfPython — CHANGES_REQUESTED** [review]

**Review by EyeOfPython — APPROVED** [review]

**Pull request merged**